core/user/_user/userProps.js

```diff
@@ -0,0 +1,14 @@
+import * as R from 'ramda'
+
+export const keysProps = {
+  props: 'props',
+  title: 'title',
+  // This key is used in Dropdown itemKey //2020
+  itemKey: 'itemKey',
+}
+
+// ====== READ
+export const getTitle = R.pipe(R.prop(keysProps.props), R.propOr('', keysProps.title))
+
+// ====== UPDATE
+export const assocTitle = R.assoc(keysProps.title)
```

Comment:
since the getter has user as context, even the setter should have the same context

Reply:
Addressed